pkg/identity/identity.go

```diff
@@ -22,6 +25,9 @@ import (
 	"github.com/spf13/viper"
 )
 
+const AES_GCM_Nonce_Size = 12
+const AES_SYMMETRICKEY_Size = 32
```

Comment:
I think this is the convention for declaring constants in Golang
```suggestion
const (
    AESGCMNonceSize     = 12
    AESSymmetricKeySize = 32
)
```